extensions/amp-consent/0.1/test/test-amp-consent.js

```diff
@@ -181,8 +178,9 @@ describes.realWin('amp-consent', {
 
     it('read promptIfUnknown from server response', function* () {
       ampConsent.buildCallback();
-      yield macroTask();
-      expect(ampConsent.consentRequired_['ABC']).to.equal(true);
+      return ampConsent.getConsentRequiredPromise_().then(isRequired => {
```

Comment:
Thank you for this! 😄 I like using promises in the tests compared to the macroTask, usually leads to my consistent tests in my opinion 😄